### PR TITLE
#2104 add responsible community to contact

### PR DESCRIPTION
**Pull request by FredrikSchaeferVitagroup**

### CODE REVIEW

**Comment by MartinWahnschaffeSymeda** [comment]
See #2104

File: sormas-api/src/main/java/de/symeda/sormas/api/contact/ContactIndexDto.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
 	public static final String DISEASE = "disease";
 	public static final String REGION_UUID = "regionUuid";
 	public static final String DISTRICT_UUID = "districtUuid";
+	public static final String COMMUNITY_UUID = "communityUuid";
```
Comment (MartinWahnschaffeSymeda) [inline sormas-api/src/main/java/de/symeda/sormas/api/contact/ContactIndexDto.java:43]: Please move these constants to the ContactJurisdictionDto. It's also used in getPrefixCaption in some places. There I'd suggest to do e.g. `I18nProperties.getPrefixCaption(ContactJurisdictionDto.I18N_PREFIX, ContactJurisdictionDto.REGION_UUID)`. ContactJurisdictionDto can use the same I18n_PREFIX as ContactIndexDto.

File: sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactDataForm.java [hunk 2]
```diff
@@ -101,7 +101,8 @@
                     fluidRowLocs(ContactDto.UUID, ContactDto.EXTERNAL_ID) +
                     fluidRowLocs(ContactDto.REPORTING_USER, ContactDto.REPORT_DATE_TIME) +
                     fluidRowLocs(ContactDto.REGION, ContactDto.DISTRICT) +
-                    fluidRowLocs(ContactDto.CASE_ID_EXTERNAL_SYSTEM, "") +
```
Comment (MartinWahnschaffeSymeda) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactDataForm.java:104]: Let's put the contact in one line with region and district instead. We are doing the same in the case in #1637

**Comment by FredrikSchaeferVitagroup** [comment]
Merged after discussion with @MartinWahnschaffeSymeda.

**Pull request merged**